addons/web_editor/static/src/js/widgets/widgets.js

```diff
@@ -1302,11 +1467,20 @@ var MediaDialog = Dialog.extend({
      * @private
      */
     _onSearchInput: function (ev) {
-        var self = this;
-        this.active.goToPage(0);
-        this.active.search($(ev.currentTarget).val() || '').then(function () {
-            self._updateControlPanel();
-        });
+        var $imageTab = this.$('#editor-media-image');
+        if ($imageTab.hasClass('show')) {
+            var inputValue = this.$('#icon-search').val();
+            this.$('.o_search_value_input').text(inputValue);
+            if ($(ev.currentTarget).val().length > 0) {
+                this.$('.o_dropdown_search').addClass('show');
+            } else {
+                this.active.search('');
+                this.$('.o_dropdown_search').removeClass('show');
+            }
+        } else {
+            this.active.goToPage(0);
+            this.active.search($(ev.currentTarget).val() || '');
+        }
```

Comment:
What is that function for ? I think the global search input has been removed ? In any case, the media dialog should not check what is inside his inner widget. Only call `this.active.doStuff()` like before.

Reply:
I see. This function enable dropdown (only in images tab) to search in local images or unsplash images.